jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "types": ["@league-of-foundry-developers/foundry-vtt-types"],
+    "moduleResolution": "node",
+    "strictNullChecks": true,
+  }
+}
```

Comment:
This doesn't seem to do anything, and this project doesn't use Typescript, what's it for?

Reply:
removed